.github/workflows/build.yml

```diff
@@ -49,23 +48,21 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
+      - id: setup-python
+        uses: actions/setup-python@v2
         with:
           python-version: 3.9
-      - name: Store installed Python version
-        run: |
-          echo "PY_VERSION="\
-          "$(python -c "import platform;print(platform.python_version())")" \
-          >> $GITHUB_ENV
       - name: Cache testing environments
         uses: actions/cache@v2
         with:
           path: ${{ env.PIP_CACHE_DIR }}
-          key: "test-${{ runner.os }}-py${{ env.PY_VERSION }}-\
-            ${{ hashFiles('**/requirements-test.txt') }}-\
-            ${{ hashFiles('**/requirements.txt') }}"
+          key: "test-${{ runner.os }}-\
+          py${{ steps.setup-python.outputs.python-version }}-\
+          ${{ hashFiles('**/requirements-test.txt') }}-\
+          ${{ hashFiles('**/requirements.txt') }}"
```

Comment:
Can we remove the quotes here?

Reply:
This should be done upstream.  I think this is something that @mcdonnnj is working on.